docs/release-notes/ReleaseNotes_17_9.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 1
+title: Release Notes 17.9
+---
+
+# Release Notes 17.9
+
+This document describes what has changed in Macrometa release 17.9.
```

Comment:
@jamesm-macrometa - I'm told there is an API call for checking the release you're on. Could you add a link to that in this paragraph?